packages/components/src/border-control/border-control-dropdown/component.tsx

```diff
@@ -65,68 +66,77 @@ const getColorObject = (
 	return colors.find( ( color ) => color.color === colorValue );
 };
 
-const getToggleAriaLabel = (
+const getToggleDescription = (
```

Comment:
I think it would be good to extract this function and write some simple tests for the various conditionals.

Then it is probably worth refactoring it with the aim of removing the branches and conditionals.

Reply:
I agree that this should all get refactored. As I was working within the context of what was here, I didn't want to change up things too much since I only wanted to provide a better description.

Reply:
It needs tidying up. I appreciate that's a big ask for this PR though. I would help but I don't have bandwidth at this time 🙇